src/frontend/src/pages/DesignReviewSummaryModal.tsx

```diff
@@ -0,0 +1,318 @@
+const Pill: React.FC<{
+  icon: React.ReactNode;
+  text: string;
+}> = ({ icon, text }) => {
+  return (
+    <Typography
+      sx={{
+        fontSize: 13,
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        paddingX: 1,
+        width: 'fit-content'
+      }}
+    >
+      {icon}
+      <Typography paddingLeft="5px">{text}</Typography>
+    </Typography>
+  );
+};
+
+// component for the DR pills that are links (zoom and docs)
+const LinkPill: React.FC<{
+  icon: React.ReactNode;
+  linkText: string;
+  displayText: string;
+}> = ({ icon, linkText, displayText }) => {
+  return (
+    <Typography
+      sx={{
+        fontSize: 13,
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        paddingX: 1,
+        width: 'fit-content'
+      }}
+    >
+      {icon}
+      <Link target="_blank" sx={{ color: 'white' }} href={linkText} paddingLeft="5px">
+        {displayText}
+      </Link>
+    </Typography>
+  );
+};
+
+// component for the member pill displaying full name
+const MemberPill: React.FC<{ user: User }> = ({ user }) => {
+  return (
+    <Box sx={{ borderRadius: '11px', backgroundColor: '#d9d9d9', width: 'fit-content', margin: '8px' }}>
+      <Typography
+        fontSize="15px"
+        paddingX="10px"
+        paddingY="1px"
+        color="#242526"
+      >{`${user.firstName} ${user.lastName}`}</Typography>
+    </Box>
+  );
+};
+
+interface StageGateModalProps {
+  wbsNum: WbsNumber;
+  modalShow: boolean;
+  handleClose: () => void;
+}
+
+export interface FormInput {
+  confirmDone: boolean;
+}
+
+// stage gate modal component (redacted loading and error for this specific case)
+const StageGateModal: React.FC<StageGateModalProps> = ({ wbsNum, modalShow, handleClose }) => {
+  const auth = useAuth();
+  const history = useHistory();
+  const toast = useToast();
+  const { mutateAsync } = useCreateStageGateChangeRequest();
+
+  const handleConfirm = async ({ confirmDone }: FormInput) => {
+    handleClose();
+    if (auth.user?.userId === undefined) throw new Error('Cannot create stage gate change request without being logged in');
+    try {
+      await mutateAsync({
+        submitterId: auth.user?.userId,
+        wbsNum,
+        type: ChangeRequestType.StageGate,
+        confirmDone
+      });
+      history.push(routes.CHANGE_REQUESTS);
+    } catch (e: unknown) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  return <StageGateWorkPackageModal wbsNum={wbsNum} modalShow={modalShow} onHide={handleClose} onSubmit={handleConfirm} />;
+};
+
+// delay wp modal component
+const DelayModal: React.FC<{ open: boolean; onHide: () => void; dr: DesignReview }> = ({ open, onHide, dr }) => {
+  const toast = useToast();
+  const [weeks, setWeeks] = useState<string>('');
+  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value === '' || parseInt(e.target.value) >= 1) {
```

Comment:
i added that so that the user can delete whatever is in the input field. if its not there, and the default is 1, the only way to change it is to use the arrow bar things that increment one at a time

Reply:
im adding smth to get around that

Reply:
Ah got it, the way it works right now with a number going into the text field works really well